src/Helpers/UtilsHelper.php

```diff
@@ -17,17 +17,30 @@
  */
 class UtilsHelper
 {
+	/**
+	 * Manifest file path getter for better loading time.
+	 *
+	 * @var array<mixed>
+	 */
+	private static $esFormsUtilsManifest = [];
+
 	/**
 	 * Return main manifest.json file.
 	 *
 	 * @return array<mixed>
 	 */
 	public static function getUtilsManifest(): array
 	{
+		if (!empty(self::$esFormsUtilsManifest)) {
+			return self::$esFormsUtilsManifest;
+		}
+
 		$sep = \DIRECTORY_SEPARATOR;
 		$filePath = \dirname(__FILE__, 2) . "{$sep}manifest.json";
 
-		return \json_decode(\implode(' ', (array)\file($filePath)), true);
+		self::$esFormsUtilsManifest = \json_decode(\implode(' ', (array)\file($filePath)), true);
```

Comment:
The `file` function will return an array, or `false` if there is a failure. I'd write this so that in the case of failure, you return an empty array. Otherwise, just decode the file.

Also, I'm wondering if you could skip the `implode` part if you'd use `file_get_contents` which returns the string.

Reply:
agreed, I have tested this and it looks better and even faster as file_get_contents is slightly faster. tnx.